sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -123,6 +124,73 @@ static inline void gnrc_sixlowpan_ctx_remove(uint8_t id)
 {
     gnrc_sixlowpan_ctx_lookup_id(id)->prefix_len = 0;
 }
+
+/**
+ * @brief   Check if a prefix matches a compression context
+ *
+ * @param[in] ctx        The compression context
+ * @param[in] prefix     IPv6 prefix
+ * @param[in] prefix_len Length of the IPv6 prefix
+ *
+ * @return    true if the prefix matches the compression context.
+ */
+static inline bool gnrc_sixlowpan_ctx_match(const gnrc_sixlowpan_ctx_t *ctx,
+                                            const ipv6_addr_t *prefix, uint8_t prefix_len)
+{
+    return (ctx != NULL) &&
+           (ctx->prefix_len == prefix_len) &&
+           (ipv6_addr_match_prefix(&ctx->prefix, prefix) >= prefix_len);
+}
+
+/**
+ * @brief   Create or update a compression context
+ *
+ * @param[in] prefixe    IPv6 prefix of the compression context
+ * @param[in] prefix_len Length of the IPv6 prefix
+ * @param[in] valid      Lifetime of the prefix in seconds
+ *
+ * @return    true if a new compression context was created or an existing context
+ *                 was updated.
+ *            false if no new context could be added
+ */
+static inline bool gnrc_sixlowpan_ctx_update_6ctx(const ipv6_addr_t *prefix, uint8_t prefix_len,
+                                                  uint32_t valid)
+{
+    gnrc_sixlowpan_ctx_t *ctx = gnrc_sixlowpan_ctx_lookup_addr(prefix);
```

Comment:
... and use
```suggestion
    if (IS_USED(MODULE_GNRC_SIXLOWPAN_CTX) {
        gnrc_sixlowpan_ctx_t *ctx = gnrc_sixlowpan_ctx_lookup_addr(prefix);
```

here. Should have the same effect, if ...